src/test/fio/fio_ceph_objectstore.cc

```diff
@@ -209,7 +209,7 @@ Job::Job(Engine* engine, const thread_data* td)
   for (uint32_t i = 0; i < td->o.nr_files; i++) {
     auto f = td->files[i];
     f->real_file_size = file_size;
-    f->engine_data = i;
+    f->engine_data = (void *)(intptr_t)i;
```

Comment:
@majianpeng i understand that `engine_data` is a `void *`. but why would you want to cast to `intptr_t` before casting to `void *`? and i'd recommend use `reinterpret_cast<void*>()` to be consistent with the rest of this source file. and to be more explicit.